crds/objectstorage.k8s.io_buckets.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: (devel)
+    controller-gen.kubebuilder.io/version: v0.4.1
```

Comment:
Is this manually generated version? I am not sure how we come up with the number.

Reply:
This is not manually set. It is automatically set by go.mod and then set by the codegenerator